### DISCUSSION TITLE
Shareservice configuration

### DISCUSSION THREAD

**Discussion by timenno** [photobooth-app/photobooth-app#115] [Q&A] [answered]
Hi, I'm quite new to this and didn't work with php that much before.

I've tried to set up the Shareservice function of my photobooth.
My server can be reached by the Photobooth app and by my phone, but on my Phone states an HTTP Error Code 500.
The PHP instance I got the following error message: 

```
PHP Fatal error:  Uncaught Exception: Unable to open database: unable to open database file in /var/www/html/uploads/dl.php:71\nStack trace:\n#0 /var/www/html/uploads/dl.php(71): SQLite3->__construct()\n#1 {main}\n  thrown in /var/www/html/uploads/dl.php on line 71, referer: android-app://com.google.android.googlequicksearchbox/
``` 

maybe someone could help me

**Comment by mgineer85** [comment]
I think the folder might have wrong permissions and cannot create the SQLite db.

**Reply by mgineer85** [reply, marked as answer]
https://stackoverflow.com/a/7670618 Pls check this

**Reply by mgineer85** [reply]
You installed the php script on an online service? Or locally on the photobooth device? Latter does not make sense .

**Reply by timenno** [reply]
Thank you, you're right it was due to wrong permissions on my apache instance.

**Reply by mgineer85** [reply]
Hi, I enhanced the documentation a little to show the differences between 3 methods to share images: https://mgrl.github.io/photobooth-docs/setup/shareservice/

Hope this helps you to find the right solution for you. I would prefer method A always as it's most convenient.
